content/doc/book/installing/index.adoc

```diff
@@ -80,16 +80,18 @@ link:https://docs.docker.com/get-started/[Getting Started, Part 1: Orientation
 and setup] page.
 
 Docker's fundamental platform and container design means that a single Docker
-image (for any given application like Jenkins) can be run on any operating
-system (e.g. macOS, Linux and Windows) which is also running Docker.
+image (for any given application like Jenkins) can be run on any supported
+operating system (e.g. macOS, Linux and Windows) or cloud service (e.g. AWS and
+Azure) which is also running Docker.
```

Comment:
The "cloud service" part seems slightly overkill/repetition to me. I mean isn't it also an "operating system" there too?

Reply:
Hmm - I'm in 2 minds about this - but I'm leaning to the side of wanting to keep it...

I can see how experienced people might find this level of detail superfluous, but this documentation also caters to people who are new to the field too and many of them may have heard very little about AWS or Azure, or cloud services in general.

I also suspect that this 'new to the field' group would constitute the greater proportion of people reading this page/section and when these people read the term 'operating system', they immediately think of the OS running on their own desktop or laptop computer. Personally, and more generally, I dislike reading content which mentions technical concepts that aren't properly explained in context (or doesn't link out to easily discoverable explanations). It detracts from the flow of reading the core content itself and diminishes the quality of the documentation.

Anyway, these are the reasons for wanting to keep this level of detail in the documentation. I will, however, remove the superfluous 'e.g.'s.